client/.eslintrc

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
We're such big fans of linters that we're including the linters by default in the generator. You can turn this off.